packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -24,6 +25,7 @@ export interface IBaseButtonState {
   menuProps?: IContextualMenuProps | null;
 }
 
+@withResponsiveMode
```

Comment:
I'd really prefer not to do this for every button. 

A better solution might be to have a "default" beakVisibility logic in contextual menu. Forcing everyone to consider this at the consumer layer results in inconsistency.